packages/create-cloudflare/templates/solid/c3.ts

```diff
@@ -62,7 +63,7 @@ const configure = async (ctx: C3Context) => {
 const config: TemplateConfig = {
 	configVersion: 1,
 	id: "solid",
-	displayName: "Solid",
+	displayName: "SolidStart",
```

Comment:
I personally like this change, but I'd like to point out:

- @IgorMinar seemed interested in this: https://github.com/cloudflare/workers-sdk/issues/3263 so I'd probably check what he thinks about the renaming

- if we rename `Solid` to `SolidStart` I feel we should give the same treatment to all other frameworks or would you like to have solid be the only one? is the rationale here that `create-solid` can create non-solid-start apps? (it is the only CLI that allows such a thing I think, but that's mostly obscured by the user in C3, no?)

- shouldn't we update our docs accordingly? (as I mentioned in #3263)

Reply:
Good point, I'll revert this change